### DISCUSSION TITLE
CogVLM 是否有 Embedding 的功能

### DISCUSSION THREAD

**Discussion by ponponon** [THUDM/CogVLM#352] [Q&A] [answered]
我想输入给 CogVLM 一个 image，然后 CogVLM 输出一个 list[float] 的向量，维度可以随意

类似 resnet50 那样

CogVLM 可以实现这一点吗？

想当向量大模型用

<img width="1558" alt="图片" src="https://github.com/THUDM/CogVLM/assets/38725104/9e9c033b-d643-48ad-9ae1-e986ccc4da61">

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
没有
